src/crimson/os/seastore/cached_extent.h

```diff
@@ -620,10 +624,16 @@ class CachedExtent
 
   /// Returns true if extent or prior_instance has been invalidated
   bool has_been_invalidated() const {
-    return !is_valid() || (is_mutation_pending() && !prior_instance->is_valid());
+    if (!is_valid()) {
+      return true;
+    }
+    if (prior_instance && !prior_instance->is_valid()) {
+      return true;
+    }
+    return false;
```

Comment:
I suppose this is for better readability, am I right?

Reply:
Seems to me the existence of `prior_instance` doesn't solely related to mutation-pending after some changes in the history.

I think this version is more accurate, if the tests are passing.

Reply:
Ah, I meant why not `return !is_valid() || (prior_instance && !prior_Instance->is_valid());`?

Reply:
updated in place.